tests/integration-test/bpf/tc_pass.bpf.c

```diff
@@ -0,0 +1,15 @@
+#include <linux/bpf.h>
```

Comment:
Let's make sure we automatically build any bytecode images for programs which we maintain in tree, just add these three programs to https://github.com/bpfd-dev/bpfd/blob/main/.github/workflows/image-build.yaml#L251

Reply:
I was thinking that would be a good idea but I wasn't sure how to build the bpf bytecode, so I thought it was a good first step to at least check in the source code.  I looked at the file you referenced, but have a few questions.  I'll ping you later.

Reply:
I've updated build_ebpf so that the integration test eBPF code gets built with `cargo xtask build-ebpf`.  However, I've opened issue #533 for the automatic building and pushing of the bytecode container files because I think there are a couple of open questions about how to do it.